moment/moment-node.d.ts

```diff
@@ -379,11 +379,17 @@ declare module moment {
         meridiem?: (hour: number, minute: number, isLowercase: boolean) => string;
         calendar?: MomentCalendar;
         ordinal?: (num: number) => string;
+        week?: MomentLanguageWeek
     }
 
     interface MomentLanguage extends BaseMomentLanguage {
         longDateFormat?: MomentLongDateFormat;
     }
+    
+    interface MomentLanguageWeek {
```

Comment:
this interface name doesn't make sense -- it doesn't have to do with localization, so it shouldn't contain the word "Language". It should just be called `MomentWeek`